boards/arm/raytac_mdbt53_db_40_nrf5340/Kconfig.defconfig

```diff
@@ -0,0 +1,101 @@
+# Raytac MDBT53-DB-40 nRF5340 board configuration
+
+# Copyright (c) 2019-2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if  BOARD_RAYTAC_MDBT53_DB_40_NRF5340_CPUAPP || BOARD_RAYTAC_MDBT53_DB_40_NRF5340_CPUAPP_NS
+
+config BOARD
+	default "raytac_mdbt53_db_40_nrf5340_cpuapp"
+
+# By default, if we build for a Non-Secure version of the board,
+# enable building with TF-M as the Secure Execution Environment.
+config BUILD_WITH_TFM
+	default y if BOARD_RAYTAC_MDBT53_DB_40_NRF5340_CPUAPP_NS
+
+if BUILD_WITH_TFM
+
+# By default, if we build with TF-M, instruct build system to
+# flash the combined TF-M (Secure) & Zephyr (Non Secure) image
+config TFM_FLASH_MERGED_BINARY
+	bool
+	default y
+
+endif # BUILD_WITH_TFM
```

Comment:
This board is not supported by TF-M (in zephyr project).
The non-secure board variant should not be run in CI.
```suggestion
```

Reply:
Customers use our modules for a variety of products. This module will not only be used on one product. Modules from other companies are also done with nRF5340, and this part is set to y. Why doesn't ours support TF-M?

Reply:
@RaytacStanley We, Nordic, only support TF-M in nRF Connect SDK. In zephyr project and TF-M itself we only support the nrf5340 and nrf9160 DKs. You can see that even Thingy53 does not support TF-M here.

We will add back those lines and your board will be supported once these commits reach the nRF Connect SDK.